CREDITS.md

```diff
@@ -71,17 +73,20 @@ Contributors to the Firecracker repository:
 * George Pisaltu <gpl@amazon.com>
 * german gomez <germangb42@gmail.com>
 * Greg Dunn <gregdunn@amazon.com>
+* Gulshan Kumar <kumargu@amazon.com>
```

Comment:
kumargu@amazon.com is a duplicate, can be fixed also in .mailmap

Reply:
Fixed.